src/content/get-started/fwe/user-input.md

```diff
@@ -13,267 +13,1082 @@ Now that you know how to manage state in your
 Flutter app, how can you let users interact
 with your app and change its state? 
 
-## Introduction to user input
+## Introduction to handling user input
 
 As a multi-platform UI framework,
 there are many different ways for users
 to interact with a Flutter app.
 The resources in this section introduce
 you to some of the common widgets used
-for enabling user interaction. 
+for enabling user interaction within your app. 
 
-Some user input types, like [scrolling][],
+Some user input mechanisms, like [scrolling][],
 have already been covered in [Layouts][]. 
 
-For an introduction to how you can change app
-state in response to user interactions,
-check out the following tutorial that
-teaches you how to build a "favorite" button:
+### A note about design systems
 
-<i class="material-symbols" aria-hidden="true">flutter_dash</i> Tutorial: [Add interactivity to your Flutter app][]
+Flutter ships with prebuilt components for two design systems as part of the SDK, 
+[Material][] and [Cupertino][].
```

Comment:
At the moment, no. We can't do that without making it overly complex for the reader because the APIs are too different. Once the framework has unified the APIs a bit more between both systems, we could consider doing that.